### ISSUE TITLE
Status of this Library

### ISSUE THREAD

**Issue by wesleytodd** [creationix/jsonparse#30]
Hey @creationix!

I was just wondering about the state of this package?  With no readme, docs, recent commits or issue resolutions, but TONS of installs, I am unsure if it is safe to use this.  Is this module recommended? Or is there a newer streaming json parser around?

**Comment by mattsgarlata** [comment]
I'm using this library fairly extensively and it's been working great for me.

**Comment by wesleytodd** [comment]
Good to hear.  What about #7, #8, #26 and #21? Are these issues for you?  They seem like important issues to get resolved.

**Comment by mattsgarlata** [comment]
It looks like #7 was fixed and was just never closed.  I just tested and it seems to work for almost all of these characters:

```
ñ á é í ó ú ¿ ¡ Ñ Á É Í Ó Ú
اثممخ فاهس هس شقلاشؤث ้ำสสนไ ไนพสก ะ้รห รห ะ้ฟร หกดด
“It’s – a… <html> test/of\char™yen¥fouth¼copy©reg®”
Ʊʬʱ̚͜ΔϢЖשسڴṝἏ†ₔ₣№⅝↨≠⌂╫░▼☺ﬁﭖ︠ﺆ漢
```

A couple characters toward the end of the last line didn't work for me.

I haven't noticed the memory leak yet.  My application isn't in production yet, so that may be why.

I don't use streams2, and rounding of longs is part JavaScript not a problem with this library.

**Comment by wesleytodd** [comment]
Cool, so #7 is resolved and #26 looks complete as well.

I think I was confusing streams 2 with the actual up-to-date streams which I believe are 3.  So that should probably be closed as wont fix or something.

Is there any intent to add docs to this module?  Or do they exist somewhere that I don't see?

I will close this issue, since that covers most of my questions, thanks @mattsgarlata!
